distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -550,26 +551,130 @@ type Column
                 op_result = self.make_op "IIF" [when_true, when_false] new_name
                 adapt_unified_column op_result common_type
 
-    ## Rounding values is not supported in database columns.
-    round : Integer -> Boolean -> Problem_Behavior -> Column | Illegal_Argument | Invalid_Value_Type
-    round self decimal_places=0 use_bankers=False on_problems=Report_Warning =
-        _ = [decimal_places, use_bankers, on_problems]
-        Error.throw <| Unsupported_Database_Operation.Error "`Column.round` is not implemented yet for the Database backends."
+    ## Creates a column from a single constant value. The value must be numeric,
+       text, or boolean.
 
-    ## Truncating values is not supported in database columns.
+       This method is not static; it must be called on an existing column. That
+       existing column is only used for its internal state; the contents of the
+       column are not used.
+
+       Arguments:
+       - value: the constant value to use for the entire column. Must be
+         numeric, text, or boolean.
+
+       > Example
+         Create a column of the value 42
+
+         column.const 42
+    const : Any -> Column
+    const self value =
+        case value.is_a Number || value.is_a Boolean || value.is_a Text of
```

Comment:
What is the reason for this requirement?

Reply:
I just put this in because I hadn't tested any other types, but I'll remove it if you don't see any problems with removing it.

Reply:
This is there because handle_const in SQLite_Type_Mapping can only handle those types. Is there a more general way to ensure that this argument is a type that can be a valid database value?

Reply:
Yeah ok I think it is fine to leave it that way, that explanation seems enough.